modules/ServerAPI/conf/ProtocolConfJSON.coffee

```diff
@@ -470,11 +470,11 @@
 				name: "None"
 				ignored: false
 			,
-				codeType: "model fit"
-				codeKind: "transformation units"
+				codeType: "compound"
+				codeKind: "scientist"
 				codeOrigin: "ACAS DDICT"
-				code: "%"
-				name: "%"
+				code: "Pharmaron"
```

Comment:
Is it intentional that this unrelated ddict value was replaced by an example compound scientist?

Reply:
Yes, this was a mistake, fixing now.